add link to 'Spacefinder' title

jlf44 opened the pull request:
Adding a link to the 'Spacefinder' title

jlf44 committed:
add link to 'Spacefinder' title

mchesterkadwell commented:
I think the logo has moved up by a few pixels as a result of this change. Now I think it looks too far up. Just a few pixels but I think it's noticeable to the eye.


![Screenshot from 2023-03-02 17-46-43](https://user-images.githubusercontent.com/5013576/222510172-ffaed73a-9705-47fb-8bdd-1045aeaaf9a2.png)
![Screenshot from 2023-03-02 17-46-17](https://user-images.githubusercontent.com/5013576/222510176-5a82b0c0-965c-4513-9965-dd9768213996.png)

jlf44 committed:
add padding to center the Spacefinder title and fix issue with tiny logo in responsive, small screen design

jlf44 commented:
I've added a commit to center the title with the link in. 
I've also added a small change so that the Uni logo does not appear weirdly small in the responsive / small screen design.
